python/gradbench/gradbench/eval.py

```diff
@@ -73,11 +75,12 @@ class SingleModuleValidatedEval:
     validator: Validator
     id: int
 
-    def __init__(self, *, module: str, validator: Validator):
+    def __init__(self, *, module: str, validator: Validator, config={}):
```

Comment:
Sorry, I missed this before; I'd also rather this go in the `start` method instead.